scripts/control_plane/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
```

Comment:
> Instead of the github URL, it should just a local file system reference to to config/default.

I made this change as part of the branch I took off this one. This is solved.

> This should an overlay in the config directory.

I took a different tack with this part and created an `aux-config` directory. Have a look at my README to see why. I didn't really understand the intended operation of the config directory before looking into this PR, but looking at how `config` is laid out, I've come to believe - 

* It is intended that all operators have components (service accounts, roles, bindings, deployments, etc.) with exactly the same names.
* These are then made 'operator-specific' only through the namePrefix of the `default` kustomization.
* Features can then be turned on/off in the remaining kustomizations, but naming should remain consistent.
* It would aid consistency and clarity (especially when upgrading operator-sdk) if we could distinguish between  pre-rolled manifests and those we have added. Hence my decision to create a new directory to deploy dependancies of k8ssandra-operator.

Let me know if you agree with my thinking on this.